scripts/utilities/check_tarball.sh

```diff
@@ -23,16 +23,21 @@ src_file=$2
 echo "Checking the source code tarball..."
 # Only packages that require a public source code are provided.
 src_list="binutils gcc gdb glibc gmp tbb libdfp libhugetlbfs kernel mpc \
-	  mpfr oprofile userspace-rcu valgrind"
+	  mpfr userspace-rcu valgrind"
 # Block all the others.
 blocked_list="amino boost expat gotools libauxv libsphde openssl paflib \
 	      python tcmalloc zlib"
 
 # Started distributing libvecpf on AT 7.0-4.
-if [[ ${at_major_version} != 6.0 ]]; then
+if [[ ${at_major_version%%.*} -gt 6 ]]; then
```

Comment:
It looks like this change is unrelated to the removal of OProfile and libpfm.  It would be better to move it to a separate commit.